src/rp2_common/hardware_clocks/include/hardware/clocks.h

```diff
@@ -269,16 +269,22 @@ typedef clock_num_t clock_handle_t;
  * frequency to be specified, and will set the clock divider to achieve the exact or higher frequency
  * achievable, with the maximum being the src_freq.
  *
- * Note: That the clock hardware only support divisors of exactly 1 or 2.0->65535.0
+ * \if rp2350_specific
+ * Note: That the RP2350 clock hardware supports divisors from 1.0->65535.0 in steps of 1/65536
  *
- * See the tables in the description for details on the possible values for clock sources.
+ * \endif
+ * \if rp2030_specific
+ * Note: That the RP2040 clock hardware only support divisors of exactly 1.0 or 2.0->65535.0 in steps of 1/256
```

Comment:
Section 2.15.3.3. in https://datasheets.raspberrypi.com/rp2040/rp2040-datasheet.pdf says "A fully featured divider divides by 1 or a fractional number in the range 2.0 to 2^24-0.01" ?

Reply:
Also, "only support" -> "only supports"